src/utils/index.js

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line import/prefer-default-export
+export function isEmpty(obj) {
+  if (typeof obj !== 'object') {
+    return undefined;
```

Comment:
I have it returning `undefined` here instead of a boolean because conceptually I felt like if we provide a non-object value to `isEmpty`, returning `false` might seen to indicate that we actually did provide an object that happens to have no keys. Returning `undefined` here makes it a little more explicit that, "hey, we can't compute the value you provided because it's not an object".